interface/requirements/local.txt

```diff
@@ -3,7 +3,7 @@
 
 coverage
 factory_boy==2.8.1
-ipython==6.1.0
+--pre ipython==6.1.0
```

Comment:
Hey @antonow cool stuff! But can you tell me why this is desirable?

Reply:
When I changed the docker base image to Ubuntu, `pip` wasn't able to find version 6.1.0 of `ipython`. I found somebody on SO saying that pre-release versions aren't displayed by default so you need to prepend `--pre` to get them.

Reply:
https://pip.pypa.io/en/stable/reference/pip_install/#install-pre

Reply:
My guess is that the Python 3.6 base image had the desire version of ipython already? We may be able to go back to using that base image depending on the response to the questions in my PR description.